lib/value-pairs/cmdline.c

```diff
@@ -340,8 +340,38 @@ vp_cmdline_parse_rekey_shift (const gchar *option_name, const gchar *value,
       return FALSE;
     }
 
-  value_pairs_transform_set_add_func
-  (vpts, value_pairs_new_transform_shift (atoi (value)));
+  value_pairs_transform_set_add_func(vpts,
+                                     value_pairs_new_transform_shift (atoi (value)));
```

Comment:
From PR perspective this was just a whitespace change, so this is an optional request, but the same value validation could be applied here too.